src/main/java/com/questhelper/helpers/quests/deathontheisle/DeathOnTheIsle.java

```diff
@@ -0,0 +1,734 @@
+		speakToGuards.addDialogStep("More options...");
+		speakToGuards.addDialogStep("Naiatli.");
+		speakToGuards.addSubSteps(climbUpFromTheatreCellar);
+
+		// varbit 11257=1 if spoken to stradius a bit
+
+		snitchToGuardsStep = new ConditionalStep(this, speakToGuards);
+		snitchToGuardsStep.addStep(not(inVilla), returnToButlerAndHeadInside);
+		snitchToGuardsStep.addStep(aroundVilla, climbFirstLooseRocksToTheatre);
+		snitchToGuardsStep.addStep(outsideVillaByLooseRocks, climbSecondLooseRocksToTheatre);
+		snitchToGuardsStep.addStep(inTheatreCellar, climbUpFromTheatreCellar);
+
+		// 42 + 45
+		talkToStradiusToEnterTheTheatre = new NpcStep(this, NpcID.STRADIUS, new WorldPoint(1472, 2925, 0), "Talk to Stradius to enter the theatre.");
+		confrontNaiatli = new NpcStep(this, NpcID.NAIATLI, new WorldPoint(1465, 2932, 0), "Confront Naiatli, attacking her when she runs away. When Clodius appears, attack him instead. Use Protect from Melee to avoid all damage.");
+		var killClodius = new NpcStep(this, NpcID.CLODIUS, new WorldPoint(1469, 2937, 0), "Kill Clodius. Use Protect from Melee to avoid all damage.");
+		var killNaiatli = new NpcStep(this, NpcID.NAIATLI, new WorldPoint(1469, 2937, 0), "Kill Naiatli. Use Protect from Melee to avoid all damage.");
+
+		talkToNaiatli = new NpcStep(this, NpcID.NAIATLI, new WorldPoint(1472, 2931, 0), "Talk to Naiatli.");
+
+		confrontNaiatli.addSubSteps(killClodius, killNaiatli);
+
+		confrontNaiatliStep = new ConditionalStep(this, confrontNaiatli);
+		confrontNaiatliStep.addStep(not(inVilla), returnToButlerAndHeadInside);
+		confrontNaiatliStep.addStep(aroundVilla, climbFirstLooseRocksToTheatre);
+		confrontNaiatliStep.addStep(outsideVillaByLooseRocks, climbSecondLooseRocksToTheatre);
+		confrontNaiatliStep.addStep(inTheatreCellar, climbUpFromTheatreCellar);
+		confrontNaiatliStep.addStep(outsideTheatre, talkToStradiusToEnterTheTheatre);
+		confrontNaiatliStep.addStep(naiatliDowned, talkToNaiatli);
+		confrontNaiatliStep.addStep(trapFailed, killNaiatli);
+		confrontNaiatliStep.addStep(trapSprung, killClodius);
+
+		/// 49
+		talkToGuardsToFinishTheQuest = new NpcStep(this, NpcID.STRADIUS, new WorldPoint(1443, 2932, 0), "Talk to Stradius and Hutza to finish the quest.");
+	}
+
+	@Override
+	public List<ItemRequirement> getItemRequirements()
+	{
+		return List.of();
+	}
+
+	@Override
+	public List<ItemRequirement> getItemRecommended()
+	{
+		return List.of();
+	}
+
+	@Override
+	public List<Requirement> getGeneralRecommended()
+	{
+		return List.of(
+			new CombatLevelRequirement(40),
+			emptyInvSlots
+		);
+	}
+
+	@Override
+	public List<Requirement> getGeneralRequirements()
+	{
+		return List.of(
+			new QuestRequirement(QuestHelperQuest.CHILDREN_OF_THE_SUN, QuestState.FINISHED),
+			new SkillRequirement(Skill.THIEVING, 34),
+			new SkillRequirement(Skill.AGILITY, 32)
+		);
+	}
+
+	@Override
+	public List<String> getCombatRequirements()
+	{
+		return List.of(
+			"2 level 49 criminals without gear"
```

Comment:
Neither seemed to have levels, and you don't actually need to attack one of them. The other doesn't give experience and it doesn't count as your actual hit on them. I'm not sure the best way to convey this, but they're basically not actually enemies.

Reply:
Maybe "2 theatrical fights"?

Reply:
Seems like you can't lose any of the fights actually. On my last playthrough, I'll try just planking to Adala too, and if that's completely safe & continues the quest like normal, I'll remove the combat requirements & any mention of protect from melee